### PR TITLE
Add overlooked Start Locations and 2 scenarios.

### PR DESCRIPTION
**Pull request by pisskop**
Addss as viable start locations the out of town locations.

Assigns no scenerio or profession limitation to them, though there is an argument to be made for limiting them.

Add Scenerios:

-Out of town -That limits the locations  of starting to one of the above
-Fled the Riots - Forces the player into a free-profession, surrounded start in a public building, default bookstore.

### CODE REVIEW

**Comment by spurdo** [comment]
Double comma on line 169 breaks it.

**Comment by pisskop** [comment]
This is dependent upon #19289 due to the swamp shack addition...

File: data/json/scenarios/scenarios.json [hunk 1]
```diff
@@ -197,5 +197,24 @@
...
+    "flags": [ "SUR_START" ]
+  },
+  {
+    "type": "scenario",
+    "ident": "isolationist",
+    "name": "Out of Town",
+    "points": 1,
+    "description": "You were outside of civilization when the bombs fell; you didn't even know the end had come, or that everyone was (un)dead.  Will this ignorance save you from the fate of the rest?",
+    "start_name": "Out of City",
+    "allowed_locs": [ "Swamp Shack", "campground", "survivalist farm", "cabin_woods" ]
```
Comment (Rivet-the-Zombie) [inline data/json/scenarios/scenarios.json]: 'Swamp' is capitalized?

**Comment by pisskop** [comment]
"id": "Swamp Shack",
https://github.com/pisskop/Cataclysm-DDA/blob/master/data/json/overmap_specials.json#L1663

Aye it is

File: data/json/start_locations.json [hunk 2]
```diff
@@ -142,5 +142,41 @@
...
+  },
+  {
+    "type" : "start_location",
+    "ident" : "campground",
+    "name" : "campgrounds",
+    "target" : "campsite_a"
+  },
+  {
+    "type" : "start_location",
+    "ident" : "survivalist farm",
```
Comment (codemime) [inline data/json/start_locations.json]: Please use an underscore here.

**Comment by codemime** [comment]
> Aye it is.

It's a someone's oversight. Should be `"swamp_shack"`.

**Pull request merged**